### PR TITLE
(maint) synced file(s) with chocolatey/.github

### PR DESCRIPTION
**Pull request by choco-sync**
synced local file(s) with [chocolatey/.github](https://github.com/chocolatey/.github).

<details>
<summary>Changed files</summary>
<ul>
<li>synced local directory <code>.templates/</code> with remote directory <code>.github/GitReleaseManager/.templates/</code></li><li>created local  <code>.github/workflows/label-sync.yml</code>  from remote  <code>.github/workflows/label-sync.yml</code></li><li>synced local  <code>.github/PULL_REQUEST_TEMPLATE.md</code> with remote  <code>.github/PULL_REQUEST_TEMPLATE.md</code></li>
</ul>
</details>

---

This PR was created automatically by the [repo-file-sync-action](https://github.com/BetaHuhn/repo-file-sync-action) workflow run [#7384433349](https://github.com/chocolatey/.github/actions/runs/7384433349)

### CODE REVIEW

**Comment by gep13** [comment]
@AdmiringWorm can I get you to put your 👀  on this PR before merging?  Thanks!

File: .github/workflows/label-sync.yml
Comment (AdmiringWorm) [inline .github/workflows/label-sync.yml:1]: Just a mention. We need to monitor that the sync is successful when merging this.

**Review by AdmiringWorm — APPROVED** [review]
Just a single comment with something to note. Otherwise this looks good to me.

File: .github/workflows/label-sync.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    steps:
+      # uncomment the uses line if additional configuration in the current
+      # repository is used.
+      #- uses: actions/checkout@v3.0.2
+      - uses: EndBug/label-sync@v2.2.0
+        with:
+          config-file: |
+            https://raw.githubusercontent.com/chocolatey/.github/main/.github/labels.yml
+          request-token: ${{ secrets.SYNC_TOKEN }} # Used when getting the config files.
```
[the remark this answers is not in the document]
Comment (gep13) [reply .github/workflows/label-sync.yml:21]: Yes, it is there at that level.

**Comment by AdmiringWorm** [comment]
I have pushed an update to delete any old labels in this repository immediately. This is fine for this repository due to the low number of issues that we have.

I'll get this merged and monitor the label synchronization and make any changes if an issue looses its label.

**Pull request merged**